### PR TITLE
[docs] Correct examples to use build-in assertions

### PR DESCRIPTION
**Pull request by helen-dikareva**
Waits for https://github.com/DevExpress/testcafe/pull/997

### CODE REVIEW

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
I know it https://github.com/DevExpress/testcafe/pull/1013#issue-192600216

File: docs/articles/documentation/test-api/actions/drag-element.md [hunk 1]
```diff
@@ -65,17 +62,16 @@ Parameter              | Type                                              | Des
...
-const isDesignSurfaceEmpty = ClientFunction(() => !getDesignSurfaceElements().length);
+const designSurfaceItems = Selector('.design-surface').find('.items');
 
 test('Drag an item from the toolbox', async t => {
-    await t.dragToElement('.toolbox-item.text-input', '.design-surface');
-
-    expect(await isDesignSurfaceEmpty()).to.be.false;
+    await t
+        .dragToElement('.toolbox-item.text-input', '.design-surface')
+        .expect(designSurfaceItems.count).ok();
```
Comment (inikulin) [inline docs/articles/documentation/test-api/actions/drag-element.md]: Here and in other places: better use `.gt(0)` for `count`

**Comment by helen-dikareva** [comment]
FPR

**Pull request merged**